docs/man_pages/project/testing/run-android.md

```diff
@@ -10,19 +10,20 @@ Start an emulator and run the app inside it | `$ tns run android --emulator [<Em
 Runs your project on a connected Android device or in a native Android emulator, if configured. This is shorthand for prepare, build and deploy. While your app is running, prints the output from the application in the console and watches for changes in your code. Once a change is detected, it synchronizes the change with all selected devices and restarts/refreshes the application.
 
 ### Options
+* `--available-devices` - Shows a list of available emulators to be started and a list of already connected devices.
```

Comment:
I agree with @yyosifov, `--available-devices` should be removed, and users should use `tns devices` instead. IF you think it would affect users, then put a `to-be-deprecated message` when `--available-devices` is used so we can remove it in a following release.